devops/openshift/database/patroni-deploy.yml

```diff
@@ -31,10 +31,6 @@ objects:
       role: master
       cluster-name: ${NAME}
       app.kubernetes.io/name: patroni
-      sessionAffinity: None
```

Comment:
So basically removing these properties allows for connection with services outside this cluster?